src/operator-manual/tracing/01-quickstart.md

```diff
@@ -0,0 +1,458 @@
+# Tracing
+
+This section illustrates how to enable tracing support of
+Policy Server.
+
+Tracing allows to collect fine grained details about policy evaluations. It can
+be a useful tool for debugging issues inside of your Kubewarden deployment and policies.
+
+## The stack
+
+We will use the following tools to enable tracing:
+
+  * [Jaeger](https://www.jaegertracing.io/): it is used to receive, store
+    and visualize trace events.
+  * [OpenTelemetry](https://opentelemetry.io/): it is used to collect
+    trace events originated by PolicyServer and forward them to Jaeger
+
+The OpenTelemetry collector will be deployed as a sidecar inside of each PolicyServer
+Pod.
+
+## Setting up a Kubernetes cluster
+
+> This section gives step-by-step instructions to create a
+> Kubernetes cluster with an ingress controller enabled.
+>
+> Feel free to skip this section if you already have a Kubernetes
+> cluster where you can define Ingress resources.
+
+We are going to create a testing Kubernetes cluster using [minikube](https://minikube.sigs.k8s.io/docs/).
+
+minikube has many backends, in this case we will use the
+[kvm2](https://minikube.sigs.k8s.io/docs/drivers/kvm2/) driver
+which relies on libvirt.
+
+Assuming `libvirtd` is properly running on your machine, issue the
+following command:
+
+```console
+minikube start --driver=kvm2
+```
+
+The command will produce an output similar to the following one:
+
+```console
+$ minikube start --driver=kvm2
+😄  minikube v1.23.2 on Opensuse-Leap 15.3
+✨  Using the kvm2 driver based on user configuration
+👍  Starting control plane node minikube in cluster minikube
+🔥  Creating kvm2 VM (CPUs=2, Memory=6000MB, Disk=20000MB) ...
+🐳  Preparing Kubernetes v1.22.2 on Docker 20.10.8 ...
+    ▪ Generating certificates and keys ...
+    ▪ Booting up control plane ...
+    ▪ Configuring RBAC rules ...
+🔎  Verifying Kubernetes components...
+    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+```
+
+Now we have to enable the Ingress addon:
+
+```console
+minikube addons enable ingress
+```
```

Comment:
This is nothing that prevents merging the request but just a comment just in case you do not know. We can install the addons in the minikube start command with the `--addons` flag. ;)

Reply:
Nice, I didn't know about that